lib/trento_web/views/v1/user_view.ex

```diff
@@ -0,0 +1,23 @@
+defmodule TrentoWeb.V1.UserView do
+  use TrentoWeb, :view
+
+  def render("index.json", %{users: users}) do
+    render_many(users, __MODULE__, "user.json")
+  end
+
+  def render("show.json", %{user: user}) do
+    render_one(user, __MODULE__, "user.json")
+  end
+
+  def render("user.json", %{user: user}) do
+    %{
+      id: user.id,
```

Comment:
The same, maybe better destructing the fields in the signature, it looks more beatiful

Reply:
done